src/data-workspace/data-details-sidebar/audit-log.js

```diff
@@ -74,56 +74,64 @@ export default function AuditLog({ item }) {
-                        <DataTableColumnHeader>User</DataTableColumnHeader>
-                        <DataTableColumnHeader>Change</DataTableColumnHeader>
-                    </DataTableRow>
-                </TableHead>
+            <div className={styles.tableWrapper}>
+                <DataTable>
+                    <TableHead>
+                        <DataTableRow>
+                            <DataTableColumnHeader>Date</DataTableColumnHeader>
+                            <DataTableColumnHeader>User</DataTableColumnHeader>
+                            <DataTableColumnHeader>
+                                Change
+                            </DataTableColumnHeader>
+                        </DataTableRow>
+                    </TableHead>
 
-                <TableBody>
-                    {audits.map((audit) => {
-                        const {
-                            modifiedBy: user,
-                            previousValue,
-                            value,
-                            created,
-                            auditType,
-                            dataElement: de,
-                            categoryOptionCombo: coc,
-                            period: pe,
-                            orgUnit: ou,
-                        } = audit
-                        const key = `${de}-${pe}-${ou}-${coc}`
+                    <TableBody>
+                        {audits.map((audit) => {
+                            const {
+                                modifiedBy: user,
+                                previousValue,
+                                value,
+                                created,
+                                auditType,
+                                dataElement: de,
+                                categoryOptionCombo: coc,
+                                period: pe,
+                                orgUnit: ou,
+                            } = audit
+                            const key = `${de}-${pe}-${ou}-${coc}-${created}`
 
-                        return (
-                            <DataTableRow key={key}>
-                                <DataTableCell>
-                                    {moment(created).format('YYYY-MM-DD HH:mm')}
-                                </DataTableCell>
-                                <DataTableCell>{user}</DataTableCell>
-                                <DataTableCell>
-                                    <div>
-                                        {auditType === 'DELETE' && (
-                                            <DeletedValue
-                                                previousValue={previousValue}
-                                            />
+                            return (
+                                <DataTableRow key={key}>
+                                    <DataTableCell>
+                                        {moment(created).format(
```

Comment:
Oh I overlooked that one in my PR: This is in server time, right? Could you use the new date utils and display the client date here?

Reply:
updated @Mohammer5 . I needed to use `fromServerDate` util function to avoid variable number of hook renders

Reply:
I think we can safely use `useClientServerDate` here as the hook never re-sets the internal state and therefore never causes a re-render (see [`use-client-server-date.js`](https://github.com/dhis2/aggregate-data-entry-app/blob/development/src/shared/date/use-client-server-date.js))

Reply:
My bad! We're using `.map` here, so my suggestion doesn't make any sense!